Button/Button.js

```diff
@@ -264,13 +265,15 @@ const ButtonBase = kind({
  *
  * @hoc
  * @memberof agate/Button
+ * @mixes agate/TooltipDecorator.TooltipDecorator
  * @mixes ui/Button.ButtonDecorator
  * @mixes spotlight/Spottable.Spottable
  * @mixes agate/Skinnable.Skinnable
  * @public
  */
 const ButtonDecorator = compose(
 	Pure,
+	TooltipDecorator({tooltipDestinationProp: 'decoration'}),
```

Comment:
Is there any reason to apply `TooltipDecorator` by default in `Button`? It could make the `Button` heavy to render.

Reply:
I will remove it and only use it where needed. Thank you!